src/components/card/card.js

```diff
@@ -40,13 +40,12 @@ export const Card = ({
             <Icon viewBox="0 0 45 45" size={25} icon={iconName} />
           </div>
         )
-        : null
-      }
+        : null}
 
     </div>
     {children ? <div className={styles['card-content']}>{children}</div> : null}
     <div className={styles.content}>
-      {date ? <p className={styles['card-date']}>{date}</p> : null}
+      {date ? <p className={styles['card-date']}>{date}</p> : <p />}
```

Comment:
why did u change it ?

Reply:
When the date is not specified, button Details is shifted to the left. To hold the button on the right side we need a block element.